neqo-transport/src/cc/classic_cc.rs

```diff
@@ -95,14 +99,24 @@ pub trait WindowAdjustment: Display + Debug {
     fn on_app_limited(&mut self);
 }
 
+#[derive(Debug)]
+struct MaybeLostPacket {
+    time_sent: Instant,
+}
+
 #[derive(Debug)]
 pub struct ClassicCongestionControl<T> {
     cc_algorithm: T,
     state: State,
     congestion_window: usize, // = kInitialWindow
     bytes_in_flight: usize,
     acked_bytes: usize,
+    /// Packets that have supposedly been lost. These are used for spurious congestion event
+    /// detection.
+    maybe_lost_packets: HashMap<u64, MaybeLostPacket>,
```

Comment:
```suggestion
    maybe_lost_packets: HashMap<packet::Number, MaybeLostPacket>,
```

Can you use the type alias here?

https://github.com/mozilla/neqo/blob/ab40a794d38306cc6430e779b3e3db1ab0e578e7/neqo-transport/src/packet/mod.rs#L53